Sources/DatadogSDKTesting/DDTest.swift

```diff
@@ -248,6 +210,64 @@ public class DDTest: NSObject {
     }
 }
 
+extension DDTest {
+    func end(status: DDTestStatus.ITR, endTime: Date? = nil) {
+        let testEndTime = endTime ?? DDTestMonitor.clock.now
+        
+        switch status {
+        case .pass:
+            span.setAttribute(key: DDTestTags.testStatus, value: DDTagValues.statusPass)
+            span.status = .ok
+        case .fail:
+            span.setAttribute(key: DDTestTags.testStatus, value: DDTagValues.statusFail)
+            suite.status = .fail
+            module.status = .fail
+            span.status = .error(description: "Test failed")
+            setErrorInformation()
+        case .skip(itr: let itr):
+            span.setAttribute(key: DDTestTags.testStatus, value: DDTagValues.statusSkip)
+            if itr { span.setAttribute(key: DDTestTags.testSkippedByITR, value: "true") }
+            span.status = .ok
```

Comment:
If I haven't missed anything that the biggest difference between using `DDTestStatus` and `ITR`.

nit:
Not a big fan of setting attributes using hardcoded `"true"` string. Maybe worth coming up with some type -> string translation. Not a blocker of course :)

Reply:
`DDTestStatus` is for public API and Obj-C compatible. `DDTestStatus.ITR` is an internal enum. ITR skip flag can't be set to true from the public API.

`true` string - my bad. Can use Bool. Fixed.